ssz-rs/Cargo.toml

```
@@ -16,18 +16,24 @@ exclude = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["serde"]
+default = ["serde", "std"]
+std = [
+    "bitvec/std",
+    "sha2/std",
+    "num-bigint/std",
+    "serde/std"
+]
 serde = ["dep:serde", "dep:hex"]
 
 [dependencies]
 thiserror = "1.0.25"
-bitvec = "1.0.0"
+bitvec = { version = "1.0.0", default-features = false, features=["alloc"]}
 ssz-rs-derive = { path = "../ssz-rs-derive"}
-sha2 = "0.9.8"
-lazy_static = "1.4.0"
+sha2 = { version ="0.9.8", default-features = false}
+lazy_static = { version = "1.4.0", features = ["spin_no_std"]}
```

I think we can try this:

```suggestion
lazy_static = { version = "1.4.0", default-features = false, features = ["spin_no_std"]}
```